lms/djangoapps/django_comment_client/tests/utils.py

```diff
@@ -47,3 +52,83 @@ def setUp(self):
         self.moderator.roles.add(Role.objects.get(name="Moderator", course_id=self.course.id))
         self.student_cohort.users.add(self.student)
         self.moderator_cohort.users.add(self.moderator)
+
+
+@override_settings(MODULESTORE=TEST_DATA_MOCK_MODULESTORE)
```

Comment:
You need to rebase on master to get the recent changes in Cale's PR. You can then delete this line, because it is the default MODULESTORE setting with ModuleStoreTestCase. Remember to delete the import.